dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/RequestItemRepository.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.rest.repository;
+
+import java.sql.SQLException;
+import java.util.UUID;
+
+import org.dspace.app.requestitem.RequestItem;
+import org.dspace.app.requestitem.service.RequestItemService;
+import org.dspace.app.rest.converter.RequestItemConverter;
+import org.dspace.app.rest.exception.RepositoryMethodNotImplementedException;
+import org.dspace.app.rest.model.RequestItemRest;
+import org.dspace.app.rest.model.hateoas.RequestItemResource;
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Bitstream;
+import org.dspace.content.Item;
+import org.dspace.content.service.BitstreamService;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.jdbc.UncategorizedSQLException;
+import org.springframework.stereotype.Component;
+
+/**
+ * Component to expose item requests.
+ *
+ * @author Mark H. Wood <mwood@iupui.edu>
+ */
+@Component(RequestItemRest.CATEGORY + '.' + RequestItemRest.NAME)
+public class RequestItemRepository
+        extends DSpaceRestRepository<RequestItemRest, String> {
+    private static final Logger LOG = LoggerFactory.getLogger(RequestItemRepository.class);
+
+    @Autowired(required = true)
+    protected RequestItemService requestItemService;
+
+    @Autowired(required = true)
+    protected RequestItemConverter requestItemConverter;
+
+    @Autowired(required = true)
+    protected BitstreamService bitstreamService;
+
+    @Autowired(required = true)
+    protected ItemService itemService;
+
+    @Override
+    public RequestItemRest findOne(Context context, String id) {
```

Comment:
What permissions are applicable here? Are item requests restricted to the current user?
I would also recommend to create ITs to verify permissions are applied correctly.